packages/@o3r/eslint-config-otter/rules/typescript/eslint.cjs

```diff
@@ -113,6 +113,7 @@ module.exports = {
       {
         'allow': [
           'warn',
+          'error',
```

Comment:
I would actually remove `warn`, `info` and `debug` instead of adding `error`

Reply:
If we want to be more strict I think we should warn our users upfront.

I have whitelisted `error` because this is what angular generates in `main.ts` for a new app:
```typescript
bootstrapApplication(AppComponent, appConfig)
  .catch((err) => console.error(err));
```
Do you think it is worth modifying the generated code to add a `// eslint-disable-next-line no-console`?

Reply:
I would modify the code to add the `// eslint-disable-next-line no-console` as explain in this [comment](https://github.com/AmadeusITGroup/otter/pull/1698#discussion_r1576302185)

Reply:
it's hard to justify enforcing a rule that we don't respect ourselves

Reply:
Yes, then we need to fix the code to follow it.

As reminder, the 2 main purposes of it is:
- ensure we are reporting issue via proper logger, not the console directly, to avoid message miss.
- help preparing the cross environment capability (SSR for example)

Reply:
I am jumping into the topic as I received multiple feedbacks that angular is too verbose, and we are adding extra boilerplate on top of generated angular code. 
IMO if the 'public' accessor is not present, we should not raise an 'error'; The 'public' is implicit.